runtime/Cpp/runtime/src/misc/Interval.h

```diff
@@ -26,59 +26,56 @@ namespace misc {
     ssize_t b;
 
     Interval();
-    explicit Interval(size_t a_, size_t b_); // For unsigned -> signed mappings.
```

Comment:
The `explicit` keyword marks a c-tor so that no automatic type conversion is applied. So when looking up a c-tor with e.g. int values that explicit want is not considered a valid match. This way we can ensure that unsigned values only go with the explicit c-tor, while signed ones use the other c-tor (which is important because of the internal handling). I wish we could avoid using negative values. From the pure semantic of that class there cannot be a negative interval, but such values denote invalid intervals and are used when sorting intervals in an interval set (which is why we cannot use something like `(size_t)-1`).